test/shared_list_sub.rb

```diff
@@ -53,9 +79,9 @@ def test_next_prev_groups
       assert_equal [li2, li3], li1.lower_items(2)
       assert_equal [], li4.lower_items
 
-      assert_equal [li1, li2], li3.higher_items
+      assert_equal [li2, li1], li3.higher_items
```

Comment:
Is it going to break existing code if the order of returned higher items is in reverse to what it used to be?